src/svg2pdf.js

```diff
@@ -1638,12 +1646,14 @@ SOFTWARE.
       xOffset = getTextOffset(transformedText, attributeState);
 
       if (visibility === "visible") {
+        var verticalAlign = attributeState.verticalAlign;
         var textRenderingMode = getTextRenderingMode(attributeState);
         _pdf.text(
           transformedText,
           textX + dx - xOffset,
           textY + dy,
           {
+            baseline: verticalAlign,
```

Comment:
This will fail when other properties as the alignment-baseline are defined through the vertical-align shorthand. We probably should only save the alignment-baseline part in AttributeState and parse the vertical-align property with a RegExp.

Reply:
In jspdf.debug.js at line 3794, it seems to cause no problems if the value of the baseline option is not supported - jspdf just does nothing and jumps into the default case assuming there is nothing to do. Or didn't I get that right?

Reply:
Yeah that's right, but if vertical-align is something like `first sub middle`, that would fall into jspdf's default case, although it supports `middle`.